### PR TITLE
[ACCESS] Improvements to Portuguese translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[CPL] Improvement to Portuguese translation

### CODE REVIEW

File: dll/cpl/access/lang/pt-PT.rc [hunk 1]
```diff
@@ -5,22 +5,22 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Teclado"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    GROUPBOX "Teclas de aderência", -1, 6, 11, 234, 62
-    LTEXT "Use as teclas de aderência se deseja usar SHIFT, CTRL, ALT ou a tecla de logotipo do ReactOS pressionando uma tecla de cada vez.",
+    GROUPBOX "Teclas presas", -1, 6, 11, 234, 62
+    LTEXT "Utilize as ""Teclas presas"" se deseja utilizar SHIFT, CTRL, ALT ou a tecla com o logotipo do Windows pressionando uma tecla de cada vez.",
```
Comment (HBelusca) [inline dll/cpl/access/lang/pt-PT.rc:9]: "Windows" ??

File: dll/cpl/access/lang/pt-PT.rc [hunk 2]
```diff
@@ -262,16 +262,16 @@ END
...
-    LTEXT "Escolha a porta para a ligação de um dispositivo de entrada alternativo.",
+    GROUPBOX "Teclas série", -1, 6, 11, 234, 92
+    LTEXT "Seleccione a porta para ligação de um dispositivo de entrada alternativo.",
           -1, 12, 20, 222, 20
-    LTEXT "&Porta serial:", -1, 12, 34, 222, 20
+    LTEXT "&Porta série:", -1, 12, 34, 222, 20
     COMBOBOX IDC_SERIAL_PORT_COMBO, 12, 45, 222, 56,
              CBS_DROPDOWNLIST | WS_VSCROLL | WS_VISIBLE | WS_TABSTOP
-    LTEXT "&Taxa de transm.:", -1, 12, 65, 222, 20
+    LTEXT "&Velocidade de transmissão:", -1, 12, 65, 222, 20
```
Comment (HBelusca) [inline dll/cpl/access/lang/pt-PT.rc:274]: Can you check whether the text zone is large enough?

File: dll/cpl/access/lang/pt-PT.rc [hunk 3]
```diff
@@ -262,16 +262,16 @@ END
...
+CAPTION "Definições para ""Teclas série"""
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Teclas seriais", -1, 6, 11, 234, 92
-    LTEXT "Escolha a porta para a ligação de um dispositivo de entrada alternativo.",
+    GROUPBOX "Teclas série", -1, 6, 11, 234, 92
+    LTEXT "Seleccione a porta para ligação de um dispositivo de entrada alternativo.",
           -1, 12, 20, 222, 20
-    LTEXT "&Porta serial:", -1, 12, 34, 222, 20
+    LTEXT "&Porta série:", -1, 12, 34, 222, 20
```
Comment (HBelusca) [inline dll/cpl/access/lang/pt-PT.rc:271]: https://pt.wikipedia.org/wiki/Interface_serial
ou "porta serial".

File: dll/cpl/access/lang/pt-PT.rc (hunk 2 above)
Comment (zecarlos1957) [reply dll/cpl/access/lang/pt-PT.rc:274]: @HBelusca  The button that gives access to this dialog is disabled, however I believe there is free space for all the text, as it has the entire width of the window

File: dll/cpl/access/lang/pt-PT.rc (hunk 3 above)
Comment (zecarlos1957) [reply dll/cpl/access/lang/pt-PT.rc:271]: Okay, well, serial is a word adopted from English brought by computer science. (in Portuguese "série")

File: dll/cpl/access/lang/pt-PT.rc (hunk 2 above)
Comment (zecarlos1957) [reply dll/cpl/access/lang/pt-PT.rc:274]: KSerialAfter
Can we expand the window a little bit here?

File: dll/cpl/access/lang/pt-PT.rc (hunk 3 above)
Comment (zecarlos1957) [reply dll/cpl/access/lang/pt-PT.rc:271]: @HBelusca ready to aprove?

**Comment by zecarlos1957** [comment]
https://www.infopedia.pt/$porta-serie
@HBelusca  Here is a dictionary for Portuguese (Portugal). Do not confuse with Brazilian Portuguese (SOUTH emisphere). Ready for approval?

**Pull request merged**